enterprise/api_client/lms.py

```diff
@@ -341,6 +341,32 @@ def get_course_details(self, course_id):
             return None
 
 
+class CourseApiClientJwt(JwtLmsApiClient):
```

Comment:
Again, a nitpick but my preference would be to end the class name with "client", as in `CourseApiJwtClient`